controller/workitemtype.go

```diff
@@ -147,6 +142,19 @@ func (c *WorkitemtypeController) List(ctx *app.ListWorkitemtypeContext) error {
 	})
 }
 
+// stripBaseWorkItemTypes finds base types (e.g. "planneritem") in a slice of
+// given work item types and returns a slice without those base types that shall
+// not appear in the UI.
+func stripBaseWorkItemTypes(wits []workitem.WorkItemType) []workitem.WorkItemType {
+	res := []workitem.WorkItemType{}
+	for _, wit := range wits {
+		if wit.ID != workitem.SystemPlannerItem {
```

Comment:
So this expression works uuid != uuid. What about `uuid.Equal(wit.ID, workitem.SystemPlannerItem)`, as you mentioned ?